scripts/strip-frameworks.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
I don't know, should it? Our other scripts don't include it, so is that an omission?